src/Visualization/Python/InterpolateVolumeData.py

```diff
@@ -0,0 +1,372 @@
+                        help="The path to one or multiple source .h5 files. "
+                        "Accepts glob wildcards.")
+
+    parser.add_argument(
+        "--target_files",
+        type=str,
+        nargs='+',
+        help="the path to where the interpolated function is written. "
+        "The number of target files must match the number of source files."
+        " When no target file is specified, the interpolated data "
+        "is written to the source file in a new volume data.")
+
+    parser.add_argument(
+        "--tensor_components",
+        type=str,
+        nargs='*',
+        help="The names of the tensors that are to be interpolated. "
+        "Accepts regular expression. "
+        "If none are specified, all tensors are interpolated.")
+
+    parser.add_argument("--grid_extents",
+                        type=int,
+                        nargs='+',
+                        default=3,
+                        help="The extents of the target grid. "
+                        "Can be different for each dimension e.g. '3 5 4'")
+
+    parser.add_argument("--grid_basis",
+                        type=str,
+                        nargs='+',
+                        choices=basis_dict.keys(),
+                        default="Legendre",
+                        help="The basis of the target grid. "
+                        "Can be different for each dimension")
+
+    parser.add_argument("--grid_quadrature",
+                        type=str,
+                        nargs='+',
+                        choices=quadrature_dict.keys(),
+                        default="GaussLobatto",
+                        help="The quadrature of the target grid. "
+                        "Can be different for each dimension")
+
+    parser.add_argument("--source_volume",
+                        type=str,
+                        nargs='+',
+                        default=["/VolumeData"],
+                        help="The name of the volume data object within the "
+                        "source file in which the data is contained")
+
+    parser.add_argument(
+        "--target_volume",
+        type=str,
+        nargs='+',
+        default=["/VolumeDataInterpolated"],
+        help="The name of the volume data object within the target "
+        "file where the data is written.")
+
+    parser.add_argument(
+        "--observation_start",
+        type=float,
+        default=-np.inf,
+        help="Disregard all observations with value before this point")
+
+    parser.add_argument(
+        "--observation_end",
+        type=float,
+        default=np.inf,
+        help="Disregard all observations with value after this point")
+
+    parser.add_argument(
+        "--observation_stride",
+        type=float,
+        default=1,
+        help="Stride through observations with this step size.")
+
+    parser.add_argument(
+        "-j",
+        "--jobs",
+        type=int,
+        default=1,
```

Comment:
Default to `None` and just forward to `Pool`, so the default number of cores is picked automatically?